src/cider/nrepl/middleware/format.clj

```diff
@@ -67,5 +72,4 @@
    "format-edn"
    {:doc "Reformats the given EDN data, returning the result as a string."
     :requires {"edn" "The data to format."}
-    :optional {"right-margin" "The maximum column width of the formatted result."}
```

Comment:
Did you check the client code as well? Seems there are references to this param there.